bindings/ruby/src/capability.rs

```diff
@@ -57,13 +57,28 @@ define_accessors!(Capability, {
     stat: bool,
     stat_with_if_match: bool,
     stat_with_if_none_match: bool,
+    stat_with_if_modified_since: bool,
```

Comment:
I don't think we need to sync them all. In fact, Ruby doesn't support all of those options. As of now, we only need `stat`/`read` and not the other `stat_with_xxx` options. We can add them back once Ruby actually supports those APIs.

Reply:
Hmm, make sense. I was silencing a few warnings. I agree with you. We can look closely later on this.